src/apps/inventory/models.py

```diff
@@ -22,6 +32,14 @@ class Product(models.Model):
         limit_choices_to={"user_type": "Seller"},
     )
     created_at = models.DateTimeField(auto_now_add=True)
+    slug = models.SlugField(unique=True)
```

Comment:
we should validate if Product.price has a positive value

Reply:
lets keep it as it is now, we will build business requirements bit later